package.json

```diff
@@ -27,21 +27,20 @@
   "dependencies": {
     "axios": "^0.20.0",
     "chai": "^4.2.0",
-    "dotenv": "^8.2.0",
     "cherio": "^1.0.0-rc.2",
     "diff": "^4.0.2",
+    "dotenv": "^8.2.0",
     "express": "^4.17.1",
     "html-to-text": "^5.1.1",
     "mocha": "^8.1.3",
-    "mongoose": "^5.10.5"
-  },
-  "devDependencies": {
-    "chalk": "^4.1.0",
-    "csv-parse": "^4.12.0",
-    "pdf-parse": "^1.1.1",
     "mongoose": "^5.10.5",
     "nodemailer": "^6.4.11",
-    "pug": "^3.0.0"
```

Comment:
I think this is needed too in the tests.

Reply:
This is used in by express, also can be used to make the email templating easier/more flexible